### ISSUE TITLE
Gird's Keyboard navigation fails for new records when there is grouping applied

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-ui-core#3252]
### Reproduction of the problem
Open the following dojo example:
http://dojo.telerik.com/ESORu

Steps to reproduce:
1. Apply filter/group by/orderby
2. Click on Add button to generate new row
3. Fill first cell data and click on tab. Here it fails to navigate to next cell instead cursor is moving to some other element. 

### Environment

* **Kendo UI version:** 2017.2.504
* **jQuery version:** x.y
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by kdikov82** [comment]
Workaround (within Edit event):

edit: function(e){
  if(e.model.isNew()){
    e.sender.current(e.container);
    e.sender.tbody.focus();
  }
}
